wiki/Contests/Official_support/en.md

```diff
@@ -14,7 +14,7 @@ tags:
 
 For other contests, email us at [tournaments@ppy.sh](mailto:tournaments@ppy.sh). We'll try to work something out.
 
-This article was last updated on July 10, 2024. For the full changelog, check out the [beatmapping contest support updates thread](https://osu.ppy.sh/community/forums/topics/1907886). For any queries or clarifications, please consult the `#tournaments` channel in the [osu! Discord server](https://discord.com/invite/ppy) or send an email to the [account support team](/wiki/People/Account_support_team) via [tournaments@ppy.sh](mailto:tournaments@ppy.sh).
+This article was last updated on September 7, 2024. For the full changelog, check out the [beatmapping contest support updates thread](https://osu.ppy.sh/community/forums/topics/1907886). For any queries or clarifications, please consult the `#tournaments` channel in the [osu! Discord server](https://discord.com/invite/ppy) or send an email to the [account support team](/wiki/People/Account_support_team) via [tournaments@ppy.sh](mailto:tournaments@ppy.sh).
```

Comment:
```suggestion
This article was last updated on 7 September 2024. For the full changelog, check out the [beatmapping contest support updates thread](https://osu.ppy.sh/community/forums/topics/1907886). For any queries or clarifications, please consult the `#tournaments` channel in the [osu! Discord server](https://discord.com/invite/ppy) or send an email to the [account support team](/wiki/People/Account_support_team) via [tournaments@ppy.sh](mailto:tournaments@ppy.sh).
```

Reply:
Is there any specific, standardized reason to switch it around? That's how both the [Tournament support page](https://osu.ppy.sh/wiki/en/Tournaments/Official_support) and this one have been rolling by thus far.

Reply:
I guess how you have it is also fine (we use this for formatting <https://en.wikipedia.org/wiki/Wikipedia:Manual_of_Style/Dates_and_numbers#Formats>). but most newly-written articles prefer the british english version that I suggested, I think

Reply:
(In reference to https://github.com/ppy/osu-wiki/pull/12107#pullrequestreview-2287295494)

Letting it open so the other peeps in the committee check it out. If we want to follow the British version, we'll also change it in the other article on this same PR.

Reply:
Will leave as is at the moment. We can change it in a future update if the standard ever changes.

Reply:
would stick w/ the same formatting the tournament support page currently uses for consistency. both can later be updated at the same time.